cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/common/constants/external/ResourceConstants.java

```diff
@@ -9,4 +9,14 @@
      * Generator Configuration Project Name
      */
     public static final String CONFIG_PROJECT_NAME = "CobiGen_Templates";
+
+    /**
+     * Latest Jar folder downloaded from Update Templates
+     */
+    public static final String DOWNLOADED_JAR_FOLDER = "/.metadata/cobigen_jars";
+
+    /**
+     * Cobigen templates folder location
+     */
+    public static final String COBIGEN_TEMPLATES_FOLDER = "/main/CobiGen_Templates";
```

Comment:
This should not be hard-coded. The user has to be asked where to extract the templates.